src/components/DressListItem/DressListItem.css

```diff
@@ -0,0 +1,29 @@
+@import url("https://fonts.googleapis.com/css?family=Raleway&display=swap");
```

Comment:
Just one more small update. Instead of using CSS, can you update this to be using a styled component? The CSS you have looks great, it's just a matter of moving it into a component in JavaScript. Check out how some of the other components are structures (Button is a good one) if you need an example of how that is set up.